packages/algob/src/lib/deployer.ts

```diff
@@ -137,4 +146,12 @@ export class AlgobDeployerReadOnlyImpl implements AlgobDeployer {
   isDefined (name: string): boolean {
     return this._internal.isDefined(name);
   }
+
+  get algodClient() {
+    return this._internal.algodClient
+  }
```

Comment:
This is good,

But seconding the comment above `this.algodClient` vs `this.algoClient`

Reply:
algoActions